src/http/routes.cc

```diff
@@ -127,7 +126,7 @@ handler_base* routes::get_handler(operation_type type, const sstring& url,
         }
         params.clear();
     }
-    return nullptr;
+    return _default_handler;
```

Comment:
please note, that you will not be protected for mandatory parameters as other handlers do.

I don't know if that's an issue, but know that it's a limitation of your current implementation.

Reply:
Thats exactly what should be provided by the default handler, a completely flexible interface